inference-engine/tests/functional/plugin/gpu/single_layer_tests/loop.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <tuple>
+#include <string>
+#include <vector>
+#include <memory>
+#include "single_layer_tests/loop.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+using namespace LayerTestsDefinitions;
+using namespace InferenceEngine;
+
+namespace {
+    static const std::vector<std::tuple<bool, int64_t, int64_t, int64_t>> static_loop_types {
+            //  GCC4.8 limitation: have to specify type of each element in list
+            //                               static_trip_count |  max | dynamic_exit | axis
+            std::tuple<bool, int64_t, int64_t, int64_t>{  true ,  5, -1, -1 },  // n_iter 5, no dynamic exit
+            std::tuple<bool, int64_t, int64_t, int64_t>{  true ,  5, -1,  1 },  // n_iter 5, const for loop with auto concatenated out
```

Comment:
lets add more tests with other axes and iteration count ..

Reply:
Add more tests. please check it again. thanks.